gradle.properties

```diff
@@ -0,0 +1,9 @@
+archivaUser=admin
+archivaPassword=adminpass123
+archivaHostId=localhost
+archivaPort=8080
+version=0.7.0-SNAPSHOT
```

Comment:
Should be 0.8.0-SNAPSHOT

Reply:
This comment is fixed and the changes result in a successful build.